macros/special-notes.cfg

```diff
@@ -1,11 +1,20 @@
 #textdomain wesnoth-Flight_Freedom
 
 # wmlindent: start ignoring
+#define SPECIAL_NOTES_PRECISION
+    _ "This unit has precision attacks, which always have a very high chance of hitting an opponent." #enddef
```

Comment:
So, I've been trying to expunge usage of `SPECIAL_NOTES` macros from this campaign, since they're deprecated; see issue #12 in this repo... adding new ones would be kind of moving backwards in that regards...

Reply:
Fixed; should close #12.

Reply:
...ok, I see, in 4e73846, got it.